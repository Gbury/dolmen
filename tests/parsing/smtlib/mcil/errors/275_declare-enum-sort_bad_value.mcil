(declare-enum-sort e (c1 _))