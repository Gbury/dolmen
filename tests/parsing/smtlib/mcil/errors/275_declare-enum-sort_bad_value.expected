File "tests/parsing/smtlib/mcil/errors/275_declare-enum-sort_bad_value.mcil", line 1, character 25-26:
1 | (declare-enum-sort e (c1 _))
                             ^
Error while parsing an enumeration declaration, read the reserved word '_',
      but expected an enumeration value; note that keywords and reserved
      words (such as '_', 'as', ...) are not valid enumeration values.
